File: 0215-kth-largest-element-in-an-array/0215-kth-largest-element-in-an-array.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> pq = new PriorityQueue<>(Collections.reverseOrder());
        int res = 0;
        for(int i=0; i<nums.length; i++) {
            pq.add(nums[i]);
        }

        while(k>0) {
            res = pq.poll();
            k--;
        }

        return res;
    }
}

File: 0215-kth-largest-element-in-an-array/0215-kth-largest-element-in-an-array.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        pq = []

        for num in nums:
            pq.append(-num)

        heapq.heapify(pq)
        res = 0
        while k > 0:
            res = -heapq.heappop(pq)
            k -= 1
        return res
